src/NzbDrone.Core/Extras/Metadata/Consumers/MediaBrowser/MediaBrowserMetadata.cs

```diff
@@ -25,76 +25,63 @@ public class MediaBrowserMetadata : MetadataBase<MediaBrowserMetadataSettings>
 
         public override string Name => "Emby (Legacy)";
 
-        public override MetadataFile FindMetadataFile(Series series, string path)
+        public override MetadataFile FindMetadataFile(Movie movie, string path)
         {
             var filename = Path.GetFileName(path);
 
             if (filename == null) return null;
 
             var metadata = new MetadataFile
                            {
-                               SeriesId = series.Id,
+                               MovieId = movie.Id,
                                Consumer = GetType().Name,
-                               RelativePath = series.Path.GetRelativePath(path)
+                               RelativePath = movie.Path.GetRelativePath(path)
                            };
 
-            if (filename.Equals("series.xml", StringComparison.InvariantCultureIgnoreCase))
+            if (filename.Equals("movie.xml", StringComparison.InvariantCultureIgnoreCase))
             {
-                metadata.Type = MetadataType.SeriesMetadata;
+                metadata.Type = MetadataType.MovieMetadata;
                 return metadata;
             }
 
             return null;
         }
 
-        public override MetadataFileResult SeriesMetadata(Series series)
+        public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFile)
         {
-            if (!Settings.SeriesMetadata)
+            if (!Settings.MovieMetadata)
             {
                 return null;
             }
 
-            _logger.Debug("Generating series.xml for: {0}", series.Title);
+            _logger.Debug("Generating movie.xml for: {0}", movie.Title);
             var sb = new StringBuilder();
             var xws = new XmlWriterSettings();
             xws.OmitXmlDeclaration = true;
             xws.Indent = false;
 
             using (var xw = XmlWriter.Create(sb, xws))
             {
-                var tvShow = new XElement("Series");
+                var movieElement = new XElement("Movie");
 
-                tvShow.Add(new XElement("id", series.TvdbId));
-                tvShow.Add(new XElement("Status", series.Status));
-                tvShow.Add(new XElement("Network", series.Network));
-                tvShow.Add(new XElement("Airs_Time", series.AirTime));
+                movieElement.Add(new XElement("id", movie.ImdbId));
+                movieElement.Add(new XElement("Status", movie.Status));
 
-                if (series.FirstAired.HasValue)
-                {
-                    tvShow.Add(new XElement("FirstAired", series.FirstAired.Value.ToString("yyyy-MM-dd")));
-                }
+                movieElement.Add(new XElement("ContentRating", movie.Certification));
```

Comment:
Not sure, but I don't think we use the Certification field (i.e. we don't save it).